### PR TITLE
Change the default port to 6419

### PR DESCRIPTION
**Pull request by joeyespo**
This addresses #123.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
     $ pip install grip
     $ cd myproject
     $ grip
-     * Running on http://localhost:5000/
```
Comment (srirams6) [inline setup.py]: @joeyespo  New port number hasn't been added here. Is this intentional?

**Comment by ssokolow** [comment]
Something to be aware of if you're not going to randomize at runtime:

http://www.speedguide.net/port.php?port=6419
http://www.linuxtv.org/vdrwiki/index.php/Svdrp

**Comment by joeyespo** [comment]
@ssokolow Nice! That's a great resource, thanks.

I imagine most people won't be sending messages to their TV these days, especially since "send to TV" technology is becoming more popular. And those who are probably won't be running grip at the same time. So think it's good otherwise? If it causes a serious conflict with anyone, we can change it in another release. (I was originally going to go with `64199` but shortened it.)

**Comment by ssokolow** [comment]
Yeah. Seems good otherwise.

**Comment by joeyespo** [comment]
Thanks for the input!

**Pull request merged**